apps/builder/app/builder/features/style-panel/sections/transforms/transform-scale.tsx

```diff
@@ -266,34 +177,29 @@ export const ScalePanelContent = (props: TransformPanelProps) => {
             description={propertySyntaxes.scaleZ}
           />
           <CssValueInput
-            key="scaleZ"
             styleSource="local"
-            property={fakeProperty}
+            property={property}
```

Comment:
<img width="255" alt="Screenshot 2024-09-19 at 11 12 51 AM" src="https://github.com/user-attachments/assets/bb3d935c-b3f2-4ebe-9cb8-31588436153e">


![sclae-none-bug](https://github.com/user-attachments/assets/dd3f1388-25cc-48a0-89a9-e95ec0ba40f0)

We need to set the `fakeProperty` here, or else it allowing user to set keywords. But when used in `scaleX` and `scaleY` keywords can't be set of just one axis

Reply:
Hm, this is not a keyword. It's some weird unit in there.

Reply:
It comes from here 
https://developer.mozilla.org/en-US/docs/Web/CSS/scale#syntax

We are setting property as `scale` directly and so the `CSSValueInput` is picking the keywords available for the property directly. But in reality we don't allow to set just one value for the scale from UI right. So used `opacity` before which has the same syntax for individual values and no keywords like `none` as such.
https://developer.mozilla.org/en-US/docs/Web/CSS/opacity#syntax

Reply:
I mean we do not pass keywords list of css input. And you are pointing to unit select. For some reason it shows keyword, but it shouldn't.

Reply:
It comes from here
https://github.com/webstudio-is/webstudio/blob/main/apps/builder/app/builder/features/style-panel/shared/css-value-input/unit-select.tsx#L50

`buildOptions` is taking keywords from the auto-generated properties list to show under it. So even if the passed `keywords` list is empty. It picks from the `UnitSelector`.

Reply:
Killed it